Sources/GPX/OAGPXDatabase.mm

```diff
@@ -365,6 +366,9 @@ - (void)load
         OAGPX *gpx = [self generateGpxItem:gpxData];
 
         // Make compatible with old database data
+        if (!gpx.creationDate)
+            gpx.creationDate = gpx.importDate;
```

Comment:
You should implement [similar](https://github.com/osmandapp/OsmAnd/blob/c37664ae270490687cf4e171c929a2cd258c62fc/OsmAnd-java/src/main/java/net/osmand/gpx/GPXUtilities.java#L1392) method to get creation date